app/controllers/students_controller.rb

```diff
@@ -10,6 +10,11 @@ def index
     @students = Student.all
   end
 
+  def dashboard
+    authorize Student
+    @students = Student.all
```

Comment:
I would guess that users are only interested in seeing active students: `Student.active`